src/test/java/org/jitsi/meet/test/base/Participant.java

```diff
@@ -86,6 +87,13 @@ public void joinConference(String roomName)
      */
     public void joinConference(String roomName, String fragment)
     {
+        // not hanguped, so not joining
+        if (!this.hungUp)
+        {
+            TestUtils.print("Not joining " + roomName + ", already joined.");
```

Comment:
how do you know that it's the given roomName that is joined ? it could be anything

Reply:
Yep, will fix that.